cloudinit/sources/DataSourceOracle.py

```diff
@@ -372,18 +298,6 @@ def get_datasource_list(depends):
 
 
 if __name__ == "__main__":
-    import argparse
-    import os
-
-    parser = argparse.ArgumentParser(description='Query Oracle Cloud Metadata')
-    parser.add_argument("--endpoint", metavar="URL",
-                        help="The url of the metadata service.",
-                        default=METADATA_ENDPOINT)
-    args = parser.parse_args()
-    sys_uuid = "uuid-not-available-not-root" if os.geteuid() != 0 else None
-
-    data = read_metadata(endpoint_base=args.endpoint, sys_uuid=sys_uuid)
-    data['is_platform_viable'] = _is_platform_viable()
-    print(util.json_dumps(data))
```

Comment:
The parser actually gets you help output in addition to --endpoint.

And you dropped the _is_platform_viable check, which seems like useful information to provide to someone running this.

Reply:
I've addressed this in the latest push.